### DISCUSSION TITLE
Obsidian Web settings dialog flashing

### DISCUSSION THREAD

**Discussion by owrede** [coddingtonbear/obsidian-web#160] [Q&A]
Hi!

I upgraded to 2.0.6, added the API key and the certificate, but I still see the "ERR" label in the Obsidian Web plugin icon. I also reset the crypto and repeated the setup.

When clicking that I see a message flashing for a milisecond. Then the dialog for posting to obsidian appears – but does not do anything.

I screenrecorded that and was able to grab a frame that shows the message that is flashing:

<img width="1914" alt="image" src="https://github.com/coddingtonbear/obsidian-local-rest-api/assets/2369810/53f192ee-7c4f-49ef-8213-088836223929">

I did set up the plugin in the options (seems fine from there → green shield).

Any hint how to debug this?

**Comment by owrede** [comment]
Here is the console log of the service worker:

[console-1706608491872.log](https://github.com/coddingtonbear/obsidian-local-rest-api/files/14095952/console-1706608491872.log)

**Comment by coddingtonbear** [comment]
I’m afraid you posted an issue to the obsidian-local-rest-api project instead of starting a discussion under the correct project (obsidian-web) so I’ve moved/converted it and now will have to ask you to answer the questions the templates will usually require you to fill in:

1. What operating system is your browser running on (windows, ubuntu, etc.)
2. What browser are you running (chrome, chromium, brave, edge, etc.)
3. What version of your browser are you running?

**Reply by owrede** [reply]
1. Operating System: MacOS 14.3
2. Running: Chrome
3. Version: 121.0.6167.139

**Reply by owrede** [reply]
I need to add:
After restarting Obsidian it seesm to work ONCE and it shows a the number of posts that contain a urls to the current site (like I am at https://heise.de/ and it shows that I have 7 links to this site in my notes). But this is a short effect. Any other site will lead to en ERR badge and once that is present it won't go away anymore.

**Reply by coddingtonbear** [reply]
OK!  In the most recent version of the extension, there's a new feature that'll provide hopefully enough logging for this problem to be understood.

Could you:

1. Reproduce the problem
2. Immediately afterward, go to the extension options panel (right-click on the extension, click "Options")
3. Then click the little bug-shaped icon in the upper-right corner
4. Make sure that the first toggle is turned on on the displayed dialog.
5. Press the "Export" button
6. Upload that file as an attachment here?

**Comment by coddingtonbear** [comment]
_That being said_ -- I wouldn't worry about anything being briefly displayed in the pop-up dialog in the first few moments after it appears.  There are a variety of things that have to be shown depending on the state of your Obsidian Web settings, what chrome permissions are available or enabled, and whatnot, and although some care is taken to not show anything but a "loading" screen until all data has been gathered from the myriad places we need to gather it, it absolutely isn't perfect and you may briefly see pop-up messages that will quickly disappear before showing you the normal form.
